src/lightning/fabric/utilities/spike.py

```diff
@@ -0,0 +1,149 @@
+import json
+import operator
+import os
+import warnings
+from typing import Any, Dict, List, Literal, Optional, Protocol, runtime_checkable, Union
+
+import torch
+from lightning_utilities.core.imports import compare_version
+
+_TORCHMETRICS_GREATER_EQUAL_1_0_0 = compare_version("torchmetrics", operator.ge, "1.0.0")
+if _TORCHMETRICS_GREATER_EQUAL_1_0_0:
+    from torchmetrics.aggregation import MeanMetric
+    from torchmetrics.wrappers import Running
```

Comment:
I suggest moving this to local import following the practices we use everywhere. To avoid #12786

Reply:
done in https://github.com/Lightning-AI/lightning/pull/18014